docs/running_psalm/configuration.md

```diff
@@ -331,7 +331,7 @@ if your application registers one or more custom autoloaders, and/or declares un
   throwExceptionOnError="[bool]"
 >
 ```
-Useful in testing, things makes Psalm throw a regular-old exception when it encounters an error. Defaults to `false`.
+Useful in testing, this makes Psalm throw a regular-old exception when it encounters an error. Defaults to `false`.
```

Comment:
Also addressed in https://github.com/vimeo/psalm/pull/3863/files#diff-7a9bf5f6e0b7ce15b91d8e22da3430c9R334

Reply:
hah, beat me to it